CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- `ClientSet::PrepareSend` and `ServerSet::PrepareSend` system sets. Backends should use these sets to add `PostUpdate` logic that needs to run before sending data on clients and servers.
```

Comment:
Maybe name them explicitly like this?
```suggestion
- `ClientSet::Disconnect` and `ServerSet::Stop` system sets. Backends should use these sets to add `PostUpdate` logic that needs to run before sending data on clients and servers.
```

Reply:
Backends can put more than just disconnect or stop in these sets. Renet has `set_connecting` for example.

Reply:
Got it! How about `PreSend`? I feel like it could be a better fit for disconnect/stop and `set_connecting`.

Reply:
`PreSend` doesn't have any meaning other than ordinality. I prefer system sets with semantic meaning where possible.

Reply:
I just would expect for "Prepare" to collect some data necessary for sending, not disconnecting/stopping :thinking: 
But I'm not a native speaker, so I'll leave it up to you. Let's apply suggestions below and merge.